src/App.js

```diff
@@ -27,12 +29,12 @@ import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Home from './pages/Home';
 import RewardDetail from './components/RewardDetail';
 
-
+// code not well formatted, try to use formatter
 function App() {
-  return (
+  return ( // why adding bg image instead of color, increases app size ?
```

Comment:
The image had texture to it in addition to color (just for beautification and failed successfully)